tests/test_util.py

```diff
@@ -142,8 +158,444 @@ def test_IsAtTrajectoryEnd_ReturnsTrue(self):
+                                                err_msg=error, verbose=True)
+
+    def test_GetLinearCollisionCheckPts_SamplingOrderMethods(self):
+        q0 = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+        q1 = [0.01, 0.02, 0.03, 0.04, 0.03, 0.02, 0.01]
+        traj = self.CreateTrajectory(q0, q1)
+
+        # Linear
+        method = 'linear'
+        checks = prpy.util.GetLinearCollisionCheckPts(self.robot, \
+                                                      traj, \
+                                                      norm_order=2, \
+                                                      sampling_order=method)
+        try:
+            # Exception can be thrown only when we try to use the generator
+            num_checks = sum(1 for x in checks)
+        except ValueError:
+            error = "Unknown sampling_order method: '" + method + "' "
+            self.fail(error)
+        except Exception, e:
+            error = 'Unexpected exception thrown: ' + str(e.message)
+            self.fail(error)
+        else:
+            pass # test passed
+
+        # Van der Corput
+        method = 'van_der_corput'
+        checks = prpy.util.GetLinearCollisionCheckPts(self.robot, \
+                                                      traj, \
+                                                      norm_order=2, \
+                                                      sampling_order=method)
+        try:
+            # Exception can be thrown only when we try to use the generator
+            num_checks = sum(1 for x in checks)
+        except ValueError:
+            error = "Unknown sampling_order method: '" + method + "' "
+            self.fail(error)
+        except Exception, e:
+            error = 'Unexpected exception thrown: ' + str(e.message)
+            self.fail(error)
+        else:
+            pass # test passed
+
+        # unknown method
+        method = 'garbage'
+        checks = prpy.util.GetLinearCollisionCheckPts(self.robot, \
+                                                      traj, \
+                                                      norm_order=2, \
+                                                      sampling_order=method)
+        try:
+            # Exception can be thrown only when we try to use the generator
+            num_checks = sum(1 for x in checks)
+        except ValueError:
+            pass # test passed, an exception was thrown
+
+        except Exception, e:
+            error = 'Unexpected exception thrown: ' + str(e.message)
+            self.fail(error)
+        else:
+            self.fail('Expected exception not thrown')
+
+
+    # ConvertIntToBinaryString()
+
+    def test_ConvertIntToBinaryString(self):
+        # The integer 6 in binary is "110"
+        binary_string = prpy.util.ConvertIntToBinaryString(6)
+        expected_string = '110'
+        error = 'The binary value ' + binary_string + \
+                ' does not equal the expected value ' + expected_string
```

Comment:
Nitpick: It would be more Pythonic to do:
```python
'The binary value {:s} does not equal the expected value {:s}'.format(binary_string, expected_string)
```